README.md

```diff
@@ -2,6 +2,8 @@
   <img src="http://www.numpy.org/_static/numpy_logo.png"><br>
 </div>
 
+[![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](http://numfocus.org)
+
```

Comment:
Sorry, that was vague. The badge seems fine. Was referring to the `http://numfocus.org` bit.